client/my-sites/theme/main.jsx

```diff
@@ -613,11 +622,13 @@ class ThemeSheet extends React.Component {
 				className="theme__sheet-primary-button"
 				href={ getUrl ? getUrl( this.props.id ) : null }
 				onClick={ this.onButtonClick }
-				primary={ isActive }
+				primary
 				target={ isActive ? '_blank' : null }
 			>
 				{ this.isLoaded() ? label : placeholder }
-				{ this.props.isWpcomTheme && this.renderPrice() }
+				<Badge type="info" className="theme__sheet-badge-beta">
+					{ this.props.isWpcomTheme && this.renderPrice() }
```

Comment:
I've noticed a fun issue with this.

If the theme we are looking at is the active one for the current site, `renderPrice()` renders an empty string.

Previously, this was ok, since we were rendering it "unwrapped".
Now, though, it's inside a `Badge`.
This is what it looks like when `renderPrice()` is empty:

<img width="190" alt="Screenshot 2021-09-28 at 16 29 46" src="https://user-images.githubusercontent.com/2070010/135118554-23097b21-962d-458f-a0f0-0fca19bd427b.png">

Reply:
Good catch! I'll take a look in a bit.

Reply:
Made updates with in f3f6e02